mmcv/utils/config.py

```diff
@@ -85,18 +85,24 @@ def _file2dict(filename):
         check_file_exist(filename)
         if filename.endswith('.py'):
             with tempfile.TemporaryDirectory() as temp_config_dir:
+                temp_config_file = tempfile.NamedTemporaryFile(
+                    dir=temp_config_dir, suffix='.py')
```

Comment:
If we use random filenames, the temp dir may not be necessary. We can just use the default temp dir.

Reply:
We had better avoid to insert the default `tmp` folder into `sys.path.`.